JustSaying.IntegrationTests/JustSaying.IntegrationTests.csproj

```diff
@@ -2,6 +2,9 @@
   <PropertyGroup>
     <TargetFramework>netcoreapp2.1</TargetFramework>
   </PropertyGroup>
+  <PropertyGroup Condition="'$(Configuration)|$(Platform)'=='Debug|AnyCPU'">
+    <NoWarn>1701;1702;CA2007;CA1054;CA1307;CA1063;CA1822;CA1052;CA1034;CA1816</NoWarn>

```

Comment:
Did you miss prefixes on 1701 and 1702?

Reply:
Hmm, I did that via the VS GUI, and those values were already there. I assumed that they were pre-existing, but it look more like it's a default generated when you write that field. Safe to remove though, so I have done so.